toolbox/util/commonUtils.go

```diff
@@ -116,8 +132,24 @@ func UpdateKeyValueInFile(file, key, value string) error {
 			}
 		}
 	}
-	if !found {
-		return fmt.Errorf("no occurrence of %s found in %s", key, file)
+	return lines, found
+}
+
+// UpdateKeyValueInFile updates in the file all occurrences of key to
+// a new value, except comments started with # or //
+func UpdateKeyValueInFile(file, key, value string) error {
+	input, err := ReadFile(file)
+	if err != nil {
+		return err
+	}
+	var found, foundToml bool
+	lines := strings.Split(input, "\n")
+	lines, found = updateKeyValueInLines(lines, key, value)
```

Comment:
should this line be an "else" to the "if file == "Gopkg.toml" ?

Reply:
no, i wanted both to run on toml